Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -92,9 +92,9 @@ public void TestLoginTimeout()
                     conn.Open();
                     Assert.Fail();
                 }
-                catch(SnowflakeDbException e)
+                catch (AggregateException e)
                 {
-                    Assert.AreEqual(SFError.REQUEST_TIMEOUT.GetAttribute<SFErrorAttr>().errorCode,e.ErrorCode);
+                    Assert.AreEqual(270007, ((SnowflakeDbException)e.InnerException).ErrorCode);

```

Comment:
Use const variable. Don't we have a list of error code for .NET driver?

Reply:
Addressed